npm-shrinkwrap.json

```diff
@@ -8746,14 +8739,39 @@
       }
     },
     "resin-corvus": {
-      "version": "1.0.0-beta.26",
-      "from": "resin-corvus@1.0.0-beta.26",
-      "resolved": "https://registry.npmjs.org/resin-corvus/-/resin-corvus-1.0.0-beta.26.tgz",
+      "version": "1.0.0-beta.27",
+      "from": "resin-io-modules/resin-corvus#v1.0.0-beta.28",
```

Comment:
Huh? How come the `version:` here is  `1.0.0-beta.27` but the `from:` is `resin-io-modules/resin-corvus#v1.0.0-beta.28`?

(Just for comparison, `package.json` says `"resin-corvus": "github:resin-io-modules/resin-corvus#v1.0.0-beta.28"`)